blinkt.go

```diff
@@ -41,6 +49,24 @@ func writeByte(val int) {
 	}
 }
 
+func convertBrightnessToInt(brightness float64) int {
+
+	if !inRangeFloat(minBrightness, brightness, maxBrightness) {
+		log.Fatalf("Supplied brightness was %#v - value should be between: %#v and %#v", brightness, minBrightness, maxBrightness)
```

Comment:
I toyed with the same question myself - fail / cap the value / use the default.  My stance was that anything other than exiting was dangerous (in a general concept sense) as the lib is effectively overriding the user's intention, they'd see a working example even though they had supplied an invalid value, i.e setting to any number > 1.0 would achieve the same output and could be perceived to be having no effect.  

In the end I opted to go with the pimoroni equivalent:

 raise ValueError("Brightness should be between 0.0 and 1.0")

on the basis that educating the user likely to lessen the prospect of misreported issues with brightness.